rosidl_runtime_py/rosidl_runtime_py/__init__.py

```diff
@@ -15,12 +15,16 @@
 from .convert import message_to_csv
 from .convert import message_to_ordereddict
 from .convert import message_to_yaml
+from .convert import slot_types_dict_from_message
+from .import_message import import_message_from_namespaced_type
 from .set_message import set_message_fields
 
 
 __all__ = [
+    'import_message_from_namespaced_type',
     'message_to_csv',
     'message_to_ordereddict',
     'message_to_yaml',
     'set_message_fields',
+    'slot_types_dict_from_message',
```

Comment:
Since functions should start with a verb how about `get_message_slot_types()`?

Reply:
renamed in https://github.com/ros2/rosidl_python/pull/33/commits/4865fb48b24933c21fc7548771f71f7de56950a3